backend/services/implementations/userService.ts

```diff
@@ -856,6 +870,8 @@ class UserService implements IUserService {
                 emergencyContactName: volunteerUser.emergencyContactName,
                 emergencyContactPhone: volunteerUser.emergencyContactPhone,
                 emergencyContactEmail: volunteerUser.emergencyContactEmail,
+                pronouns: volunteerUser.pronouns ?? "",
```

Comment:
Could we clear up this and make pronouns required as well?
```suggestion
                pronouns: volunteerUser.pronouns ??,
```

Reply:
The suggested change isnt valid, I need an expression there, do you mean just remove the ?? operator?

Reply:
> do you mean just remove the ?? operator?
Yes, that was what I was trying to suggest.